### ISSUE TITLE
Obsolete add-on packages

### ISSUE THREAD

**Issue by vholer** [OpenNebula/packages#122]
Since addons were merged into the main OpenNebula packages, they should obsolete the add-on packages not to block the upgrade process and uninstall them automatically. E.g.,

```
---> Package opennebula-sunstone.noarch 0:5.10.1-2.el7 will be updated
---> Package opennebula-sunstone.noarch 0:5.10.2-1.el7 will be an update
--> Finished Dependency Resolution
Error: Package: opennebula-addon-markets-5.10.1-2.el7.noarch (@opennebula)
           Requires: opennebula = 5.10.1
           Removing: opennebula-5.10.1-2.el7.x86_64 (@opennebula)
               opennebula = 5.10.1-2.el7
           Updated By: opennebula-5.10.2-1.el7.x86_64 (opennebula)
               opennebula = 5.10.2-1.el7
```
